priv/templates/phx.gen.live/core_components.ex

```diff
@@ -98,8 +111,8 @@ defmodule <%= @web_namespace %>.CoreComponents do
         id="server-error"
         kind={:error}
         title=<%= maybe_heex_attr_gettext.("Something went wrong!", @gettext) %>
-        phx-disconnected={show(".phx-server-error #server-error")}
-        phx-connected={hide("#server-error")}
+        phx-disconnected={show(".phx-client-error #client-error") |> JS.remove_attribute("hidden")}
+        phx-connected={hide("#client-error") |> JS.set_attribute({"hidden", ""})}
```

Comment:
As far as I know this also works and it is more succinct:

```suggestion
        phx-connected={hide("#client-error") |> JS.set_attribute("hidden")}
```

Reply:
it does not :(
https://github.com/phoenixframework/phoenix_live_view/blob/698950990440551cef8ab2b85fae32a86a4e7779/lib/phoenix_live_view/js.ex#L727-L739

Reply:
Gah, I remember now why, thank you. I guess another option is to set and remove the attribute inside show() and hide() functions. I assume this will be useful in more than one place?

Reply:
possibly

Reply:
I'll try this tomorrow and also look over everything again. Tailwind v4 is merged now.

Reply:
it doesn't work, because show runs inside an animationframe and set_attribute doesn't, so the selector we pass to show does not get any results, as the phx- classes are not set yet, apparently. So I'd leave it as is for now....